src/main/java/games/strategy/engine/data/GameMap.java

```diff
@@ -170,29 +154,6 @@ protected void addTerritory(final Territory t1) {
     m_territoryLookup.put(t1.getName(), t1);
   }
 
-  public void removeTerritory(final Territory t1) {
```

Comment:
To rule out reflection, I searched the entire workspace for occurrences of `GameMap.class` and `"remove"` and came up with zero hits.  `GameParser` is responsible for _adding_ territories to the map, but there doesn't appear to be any other class that tries to dynamically change a map's territories.

Searching through the history, it appears the last reference to this method was removed in 5592b80: the now-deleted `g.s.engine.data.changefactory.AddGridGameMapChange` class used to call it.  So sounds like a grid game artifact.